.gitignore

```diff
@@ -33,3 +33,10 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
+config/application.yml
+.idea/
+test/
```

Comment:
setup endline 
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files